src/App.jsx

```diff
@@ -1,8 +1,123 @@
 import "./App.css";
 import Todo from "./components/Todo";
+import { useEffect, useState } from "react";
+import { nanoid } from "nanoid";
 
 function App() {
-    return <div className="main bg-lime-400"></div>;
+    const [todos, setTodos] = useState(() => {
```

Comment:
This is to much code to be passed into useState hook as arguments, I think its better if you save this code into a function and call it into useState